cmake/developer_package/IEDevScriptsConfig.cmake

```diff
@@ -39,6 +39,10 @@ set_ci_build_number()
 
 include(target_flags)
 
+if(WIN32 AND AARCH64 AND OV_COMPILER_IS_CLANG)
+    message(FATAL_ERROR "For Windows on ARM, Clang compiler is not supported at the moment, please use MSVC as compiler")
+endif()
```

Comment:
I don't think we need such global error, maybe is it enough to disable CPU plugin in this case?

Reply:
The compiler check is moved to CMakeLists.txt in CPU plugin directory.